sympy/printing/tests/test_llvmjit.py

```diff
@@ -0,0 +1,45 @@
+
+from sympy.external import import_module
+
+
+if import_module('llvmlite'):
+    import sympy.printing.llvmjitcode as g
+else:
+    disabled = True
```

Comment:
Actually it looks like it's for the SymPy test runner. I don't know if this signals py.test as well (I didn't see anything in the py.test docs, but I didn't look to hard either).